docker-compose.yml

```diff
@@ -24,9 +24,22 @@ services:
       - tools
     networks:
       - recipe-app-network
+  cache:
+    image: redis:7.0
```

Comment:
I'd use a more specific version - `redis:7.0.12`. Going even further - I'd try a `redis:7.0.12-apline` first. Alpine is a super lightweight linux distribution, but because of this some feature might not work - that's why we need to test it before going with it.